content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst

```diff
@@ -2,58 +2,117 @@
+
+  .. example::
+    In `WH/Stock/Zone A/Refrigerator 1`, "Refrigerator 1" is the location name, "Zone A" is the
+    parent location, and everything before it is the path showing where this spot is within the
+    warehouse.
+
+Additional information
+----------------------
+
+In addition to the required fields above, configure the following location fields to ensure the
+location serves its intended purpose in the database:
+
+- :guilabel:`Location Type`: from the drop-down menu, choose :guilabel:`Vendor Location`,
+  :guilabel:`View`, :guilabel:`Internal Location`, :guilabel:`Customer Location`,
+  :guilabel:`Inventory Loss`, :guilabel:`Production`, or :guilabel:`Transit Location` to categorize
+  the location. For details on each location type, refer to the :ref:`Location Types section
+  <inventory/warehouses_storage/location-type>`.
+- :guilabel:`Storage Category`: only available with the :ref:`Storage Categories
+  <inventory/warehouses_storage/storage-category>` feature enabled in :menuselection:`Inventory app
+  --> Configuration --> Settings`.
+- :guilabel:`Company`: the company that the location belongs to.
+- :guilabel:`Is a Scrap Location?`: tick this box to allow for scrapped/damaged goods to be stored
   in this location
-- :guilabel:`Is a Return Location?`: check this box to allow products to be returned to this
+- :guilabel:`Is a Return Location?`: tick this box to allow products to be returned to this
   location
-- :guilabel:`Barcode`: the barcode number assigned to the location
-- :guilabel:`Removal Strategy`: the :ref:`strategy <inventory/routes/strategies/removal>` for how
-  items should be taken from inventory
+- :guilabel:`Barcode`: used with the *Barcode* app, type the barcode to :ref:`identify actions
+  <barcode/setup/location>` at this location when scanned.
+- :guilabel:`Replenish Location`: used for :doc:`configuring routes <use_routes>`, tick this
+  checkbox to set the location as a destination for receiving products from *Buy*, *Manufacture*, or
+  other procurement routes, ensuring products are correctly supplied to the warehouse.
 
-.. image:: use_locations/new-location-form.png
+.. image:: use_locations/new-location.png
    :align: center
    :alt: The form for creating a new location.
 
-Create location hierarchies
-===========================
+Cyclic counting
+---------------
+
+To schedule regular inventory counts at this location, set the :guilabel:`Inventory Frequency
+(Days)` field to the desired interval. By default, it is `0` (no scheduled counts). For example,
+setting this field to `30` schedules a count every thirty days. For more specifics on setting up and
+using this feature, refer to the :doc:`Cycle Counts documentation <cycle_counts>`.
+
+The :guilabel:`Last Effective Inventory` field displays the date the last inventory count at this
+location occurred. When scheduled inventory counts are enabled, the :guilabel:`Next Expected
+Inventory` field displays the date of the next inventory count.
```

Comment:
Do you think it's vital to mention that these fields are not modifiable? Just a suggestion...

Reply:
Do you think it's implied by the word "displayed"?